app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -18,7 +18,10 @@ def add_identity_to_log(candidate_id = current_candidate&.id)
     end
 
     def check_cookie_preferences
-      @google_analytics_id = ENV.fetch('GOOGLE_ANALYTICS_APPLY', '') if cookies['consented-to-apply-cookies'].eql?('yes')
+      if cookies['consented-to-apply-cookies'].eql?('yes')
+        @google_analytics_id = ENV.fetch('GOOGLE_ANALYTICS_APPLY', '')
+        @google_tag_manager_id = ENV.fetch('GOOGLE_TAG_MANAGER_APPLY', '')
```

Comment:
same question here around .env files and showing this value on the cookies page

Reply:
@cwrw I can't see any additional cookies being set on my development environment when I include the test GTM code so I'm not sure there's anything to add here. Will re-check in QA.